src/Cake.Common/Tools/DNX/Run/DNXRunSettings.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.DNX.Run
+{
+    /// <summary>
+    /// Contains settings used by <see cref="DNXRunner"/>
+    /// </summary>
+    public class DNXRunSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets the project dir to be used (see dnx --project option)
+        /// </summary>
+        public string Project { get; set; }

```

Comment:
Checking latest dnx source
``` 
--project|-p <PATH>
Path to the project.json file or the application folder.
Defaults to the current folder if not provided.
```
As it's either an folder or file you should adjust xml doc text to reflect that.

As it's either an FilePath or DirectoryPath then it should probably be an [Path](http://cakebuild.net/api/cake.core.io/6fd7085d) instead of string.